integration-tests/environments/react-native/package.json

```diff
@@ -50,7 +56,8 @@
     "test:ios": {
       "command": "npm run common",
       "dependencies": [
-        "pod-install"
+        "pod-install",
```

Comment:
I was looking at the `pod-install` script and noticed there are no `output`s defined.  Maybe we should define them to make local testing faster